modules/files/files_test.go

```diff
@@ -24,6 +24,21 @@ func TestFileExists(t *testing.T) {
 	assert.False(t, FileExists("/not/a/real/path"))
 }
 
+func TestDirExistsE(t *testing.T) {
+	t.Parallel()
+
+	expectedError := new(os.PathError)
+	currentFile, err := filepath.Abs(os.Args[0])
+	require.NoError(t, err)
+
+	exists1, err := DirExistsE(currentFile)
```

Comment:
Will `os.Args[0]` be the path to the executable (file) or a directory? If it's a file, this check doesn't make much sense...

Reply:
I removed this test and method